tensorflow/contrib/avro/python/parse_avro_record.py

```diff
@@ -0,0 +1,476 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import os
+import re
+
+from tensorflow.python.eager import context as _context
+from tensorflow.python.eager import execute as _execute
+
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import sparse_tensor
+from tensorflow.python.framework import tensor_shape
+
+from tensorflow.python.framework import load_library
+from tensorflow.core.framework import attr_value_pb2 as _attr_value_pb2
+from tensorflow.core.framework import op_def_pb2 as _op_def_pb2
+from tensorflow.core.framework import types_pb2 as _types_pb2
+
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops.parsing_ops import _features_to_raw_params, _prepend_none_dimension, VarLenFeature, \
+    SparseFeature, FixedLenFeature, FixedLenSequenceFeature, _construct_sparse_tensors_for_sparse_features
+
+# HACK: Change to tensorflow.contrib.avro when tensorflow/contrib/__init__.py
```

Comment:
Is this a TODO? Make it # TODO(fraudies) if so. I'm also unclear on what this is saying.

Reply:
Removed